src/components/KymaModules/ManagedWarnings.tsx

```diff
@@ -0,0 +1,100 @@
+import { Button, MessageBox } from '@ui5/webcomponents-react';
+import { useTranslation } from 'react-i18next';
+
+type ShowManagedBox = {
```

Comment:
Not quite sure here, naming is hard.
```suggestion
type ManagedBoxProps = {
```

If you go in this direction, please also change appropriate prop names elsewhere.